catroid/src/main/res/layout/activity_project_spritelist_item.xml

```diff
@@ -61,9 +62,54 @@
 
             <include layout="@layout/activity_project_sprite_title_details_view" />
 
-        </RelativeLayout >
-    </LinearLayout >
+        </RelativeLayout>
+
+        <RelativeLayout
+            android:id="@+id/spritelist_group_item_background"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:longClickable="false">
+
+            <LinearLayout
```

Comment:
Wouldn't it be easier to just have one ImageButton, and set the imageResource rather than having to views ?